examples-gallery/beginner/plot_wheel_on_bumpy_road.py

```diff
@@ -0,0 +1,401 @@
+r"""
+Quarter Car Model on a Bumpy Road
+=================================
+
+Objective
+---------
+
+- Show on a simple example how to simultaneously optimize free
+  parameters and unknown trajectories of a system.
+- Show how to use an additional state variable to get the acceleration of a
+  body into the objective function.
+- Show how to avoid possible unwanted unknown trajectories (here
+  :math:`\dfrac{d^2}{dt^2}x_{car}`).
+
+
+Introduction
+------------
+
+A quarter car model must move from a starting point to a final point on a bumpy
+road. The body (sprung mass) is connected to the wheel (unsprung mass) on the
+road by a linear spring and a damper.
+Body and wheel are modeled by particles. Movement is in the X/Z plane. Gravity
+points in the negative Z direction.
+
+I took the tire stiffness from here:
+
+https://kktse.github.io/jekyll/update/2021/07/18/re71r-255-40-r17-tire-vertical-stiffness.html
```

Comment:
The vertical tire stiffness for an automobile car is selected as 250,000 N/m based on `an experiment <https://kktse.github.io/jekyll/update/2021/07/18/re71r-255-40-r17-tire-vertical-stiffness.html>`_.

Don't use "I" in examples.